priv/worker.py

```diff
@@ -33,7 +49,7 @@
     year = {2016},
     publisher = {GitHub},
     journal = {GitHub repository},
-    howpublished = {\url{https://github.com/openai/gym-http-api}}
+    howpublished = {url{https://github.com/openai/gym-http-api}}
```

Comment:
hm, this is strange, I'll have to take a deeper, ``\\u`` will not work, but if we can't find an easy solution we can also use it without the ``url`` command.